cfn-resources/test/e2e/utility/aws_helper.go

```diff
@@ -120,14 +122,15 @@ func waitForStackDeleteComplete(svc *cfn.Client, stackID string) (*cfn.DescribeS
 		switch statusStr {
 		case "DELETE_COMPLETE":
 			return resp, nil
-		case "DELETE_FAILED":
+		case "DELETE_FAILED", "ROLLBACK_COMPLETE":
 			return nil, fmt.Errorf("stack status: %s : %s", statusStr, *resp.Stacks[0].StackStatusReason)
 		}
-		time.Sleep(3 * time.Second)
+		time.Sleep(2 * time.Second)
```

Comment:
given the comment above I would reuse the const everywhere

Reply:
added `stackStatusWait` const and re-used